docs/source/node-add-troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting - Adding or Upgrading Indy Nodes
+Many things can go wrong while adding or upgrading nodes on an existing Indy network and this guide will cover symptoms and issues encountered and some steps you might take to recover from those. The steps listed are likely just possible remedies to the listed issues. Feel free to add more remedies or issues if you don't see your's included here. As bugs are fixed, the issues noted below might not occur any more, or might have a different remedy.
+
+## Adding a Node
+This section covers troubleshooting the addition of a node to a network.  This can occur either as part of an upgrade (e.g. the 20.04 upgrade) or as part of a new node being added to an existing network.
+
+### Symptom 1 - Node is unresponsive  
+- Cause #1 - Node is performing catchup. (Large Network)
```

Comment:
Given the discussion on the Indy Contributor’s call this week (2023.09.26), should the instructions not always say to have a “recent” copy of the Indy Node data folder for the network, so the “long catchup” is not an issue?

When upgrading, the advice can be to use the node’s own data folder.

When adding a new node, include a step where the node operator’s technical team provides a data folder from another node.

Reply:
Wade has not experienced the issue, it seems, so I wondered whether "every time" is appropriate for all networks.  Until we figure out why this is only happening for me, maybe it's okay to leave this as is?
"Own" data folder is a great idea for the upgrade scenario.